src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,28 @@
-export const TodoFilter = () => (
+import { SELECT } from '../../types/SELECT';
+
+type Props = {
+  onChangeSelect: (value: SELECT) => void,
+  value: string,
+  onSearchValue: (value: string) => void,
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  onChangeSelect,
+  onSearchValue,
+  value,
+}) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
+        <select
+          data-cy="statusSelect"
+          onChange={(e) => {
+            onChangeSelect(SELECT[e.target.value as keyof typeof SELECT]);
+          }}
+        >
+          {Object.entries(SELECT).map(item => {
+            return <option value={item[0]}>{item[1]}</option>;
```

Comment:
Add a 'key' prop to the <option> element to comply with React's list rendering requirements.